src/XRP/default-xrp-client.ts

```diff
@@ -380,6 +380,28 @@ class DefaultXRPClient implements XRPClientDecorator {
     } // end for
     return payments
   }
+
+  /**
+   * Retrieve the payment transaction corresponding to the given transaction hash.
+   *
+   * Note: This method can return transactions that are not included in a fully validated ledger.
+   *       See the `validated` field to make this distinction.
```

Comment:
This is the validated field in the retrieved transaction?

Worth adding a `@see/@link` tag to link to XRPL documentation?

Reply:
So my only hesitation to linking to the XRPL documentation is that `validated` is actually a derived field that is calculated when a response to the `tx` JSON API call is assembled.  (OR when a response to the gRPC equivalent is assembled, which follows the JSON as closely as possible).  There is xrpl.org documentation for the `tx` command, but it might be confusing to be linking to a different API from the SDK. Instead, adding an `@link XRPTransaction` in the return, and the documentation there links to the ledger status documentation https://xrpl.org/ledgers.html#open-closed-and-validated-ledgers